EarlGrey/Core/GREYElementInteraction.m

```diff
@@ -97,12 +97,13 @@ - (instancetype)inRoot:(id<GREYMatcher>)rootMatcher {
  *          seconds, a timeout error will be produced and no elements will be returned.
  */
 - (NSArray *)matchedElementsWithTimeout:(NSTimeInterval)timeout error:(__strong NSError **)error {
+  NSParameterAssert(error);
```

Comment:
nit: also assert timeout param: i.e. `timeout > 0` as NSTimeInterval is double and a negative value can be passed in.